stable/jenkins/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,11 @@
+{{- if .Values.rbac.install }}
```

Comment:
@mgoodness has come up with a nice approach for RBAC. We've started suggesting following his example. I'd be great if you did too. See:

https://github.com/kubernetes/charts/pull/1504/files
https://github.com/mgoodness/charts/tree/master/stable/kube2iam